go/vt/mysqlctl/xtrabackupengine.go

```diff
@@ -277,7 +277,7 @@ func (be *XtrabackupEngine) executeFullBackup(ctx context.Context, params Backup
 	if err != nil {
 		return BackupUnusable, vterrors.Wrapf(err, "cannot JSON encode %v", backupManifestFileName)
 	}
-	if _, err := mwc.Write([]byte(data)); err != nil {
+	if _, err := mwc.Write(data); err != nil {
```

Comment:
This code is going to be called regardless of s3 or other storage backend. Why this change?

Reply:
`data` is a already a slice of bytes